contract/contracts/interfaces/DistributorInterfaceV2.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.6.0;
+
+import "@chainlink/contracts/src/v0.6/ChainlinkClient.sol";
```

Comment:
Do we still need this chainlink contract?

Reply:
Forgot to remove it. Thank you.